packages/nimble-components/src/menu/template.ts

```diff
@@ -0,0 +1,30 @@
+import {
+    children,
+    html,
+    slotted,
+    type ViewTemplate
+} from '@microsoft/fast-element';
+import type { FoundationElementTemplate } from '@microsoft/fast-foundation';
+import type { Menu } from '.';
+
+/* eslint-disable @typescript-eslint/indent */
+// prettier-ignore
+export const template: FoundationElementTemplate<
+ViewTemplate<Menu>
+> = () => html`
+    <template
+        slot="${x => {
+            if (x.slot) {
+                return x.slot;
+            }
+            return x.isNestedMenu() ? 'submenu' : '';
```

Comment:
returning `void 0` (which is just `undefined`) removes the slot attribute which is a different outcome than setting a slot to empty string: https://github.com/microsoft/fast/blob/archives/fast-element-1/packages/web-components/fast-foundation/src/menu/menu.template.ts#L15

Is there a reason for the effective change? Can we just return `undefined` which was the original behavior?